Rates to risk (#1654)
ihmeuw/vivarium_research: docs/source/models/causes/neonatal/index.rst

Modeling Strategy
+++++++++++++++++

The neonatal death model requires only the probability of death (aka "mortality risk") for the early and late neonatal time periods. Rather than using GBD mortality rates and converting them into probability of deaths, we will use mortality risk as direct input data into our model. We will calculate mortality risk input data as age-specific death counts divided by live birth counts from GBD.

Note that this strategy does not require any conversion between rates to probabilities NOR does it require any scaling to the duration of the age group. The mortality risk calculated as described below already represents the probability of dying within a neonatal age group and can be used directly as such in the simulation.

> To avoid confusion with mortality *rates* (typically referred to as the all-cause mortality rate, ACMR, or cause-specific mortality rates, CSMRs), we will refer to mortality *risk* as AMR (all mortality risk) and CMR (cause mortality risk), where:

Reviewer: pretty subtle!  Have you ever seen something "AMRi" to emphasize the "R**i**sk" rather than R**a**te?

.. math::

  AMR_{ENN} = \frac{\text{deaths due to all causes in the ENN age group}}{\text{live births}}

>   AMR_{LNN} = \frac{\text{deaths due to all causes in the LNN age group}}{\text{live births} - \text{deaths due to all causes in the ENN age group}}

Reviewer 2: In the "Data values and sources" table below, you refer to these as `enn_amr` and `lnn_emr`. I think we should pick one of the two notations and stick with it. If you want to use the notation here, it might also be good to make the font consistent with the other uses of "AMR" and "CMR" by doing `text{AMR}_\text{ENN}` and `text{AMR}_\text{LNN}`.

and 

.. math::

  CMR_{ENN} = \frac{\text{deaths due to cause C in the ENN age group}}{\text{live births}}

  CMR_{LNN} = \frac{\text{deaths due to cause C in the LNN age group}}{\text{live births} - \text{deaths due to all causes in the ENN age group}}

> Note that this strategy was updated in May of 2025 from a prior strategy of converting GBD mortality rates to probabilities. `The pull request that updated this strategy can be found here for reference. <https://github.com/ihmeuw/vivarium_research/pull/1654>`_ This strategy update was pursued following verification and validation issues in neonatal mortality and an exploration of potential solutions in model runs 6.1 through 6.4. Ultimately, a change from mortality rates to mortality risk was preferred given that it is the more policy relevant measure in the context of neonates and accurately apportioning person time alive within the neonatal age group given the input data available to us was a challenge we judged to be unnecessary.

Reviewer 2: ```suggestion
Note that this strategy was updated in May of 2025 from a prior strategy of converting GBD mortality rates to probabilities. `The pull request that updated this strategy can be found here for reference. <https://github.com/ihmeuw/vivarium_research/pull/1654>`_ This strategy update was pursued following verification and validation issues in neonatal mortality and an exploration of potential solutions in model runs 6.1 through 6.4. Ultimately, a change from mortality rates to mortality risk was preferred given that it is the more policy relevant measure in the context of neonates, and accurately apportioning person time alive within the neonatal age group given the input data available to us was a challenge we judged to be unnecessary.
```
Merely adding a comma to help with readability.

> The calculation of :math:`\text{AMR}_i` is a bit complicated, however. We begin with a population AMR and use the LBWSG PAF to derive a risk-deleted AMR to which we can then apply the relative risk of LBWSG matching any risk exposure level.  Mathematically this is achieved by the following formula:

Reviewer 2: ```suggestion
The above AMRs are population-level probabilities.
For our simulation, each simulant :math:`i` needs to be assigned an individual-level all-cause mortality risk :math:`\text{AMR}_i`, which we calculate as follows.
We begin with the population AMR and use the LBWSG PAF to derive a risk-deleted AMR to which we can then apply the relative risk of LBWSG matching any risk exposure level.  Mathematically this is achieved by the following formula:
```
We need to define `AMR_i` before we explain how to calculate it.

.. math::
    \begin{align*}
    \text{AMR}_{\text{BW},\text{GA}} &= \text{AMR} \times (1 - \text{PAF}_{\text{LBWSG}}) \times \text{RR}_{\text{BW},\text{GA}},
    \end{align*}

> where :math:`\text{AMR}_{\text{BW},\text{GA}}` is the all-cause mortality rrisk for a population with birth weight :math:`\text{BW}` and gestational age :math:`\text{GA}`, :math:`\text{AMR}` is the all-cause mortality risk for the total population, :math:`\text{PAF}_{\text{LBWSG}}` is the population attributable fraction for LBWSG, and :math:`\text{RR}_{\text{BW},\text{GA}}` is the relative mortality risk for a specific birth weight :math:`\text{BW}` and gestational age :math:`\text{GA}`.

Reviewer 2: ```suggestion
where :math:`\text{AMR}_{\text{BW},\text{GA}}` is the all-cause mortality risk for a population with birth weight :math:`\text{BW}` and gestational age :math:`\text{GA}`, :math:`\text{AMR}` is the all-cause mortality risk for the total population in one of the neonatal age groups (i.e., :math:`\text{AMR}` equals :math:`\text{AMR}_\text{ENN}` or :math:`\text{AMR}_\text{LNN}` as defined above), :math:`\text{PAF}_{\text{LBWSG}}` is the population attributable fraction for LBWSG, and :math:`\text{RR}_{\text{BW},\text{GA}}` is the relative mortality risk for a specific birth weight :math:`\text{BW}` and gestational age :math:`\text{GA}`.
```
Not sure if this is the best way to do it, but I'm trying to make it more clear that the AMR in this equation is the same as (one of) the AMRs we defined previously.

Also, minor typo fix: rrisk -> risk

Note that this strategy does not require any conversion between rates to probabilities NOR does it require any scaling to the duration of the age group. The mortality risk calculated as described below already represents the probability of dying within a neonatal age group and can be used directly as such in the simulation.

To avoid confusion with mortality *rates* (typically referred to as the all-cause mortality rate, ACMR, or cause-specific mortality rates, CSMRs), we will refer to mortality *risk* as ACMRisk (all-cause mortality risk) and CSMRisk (cause-specific mortality risk), where:

.. math::

  ACMRisk_{ENN} = \frac{\text{deaths due to all causes in the ENN age group}}{\text{live births}}

  ACMRisk_{LNN} = \frac{\text{deaths due to all causes in the LNN age group}}{\text{live births} - \text{deaths due to all causes in the ENN age group}}

and 

.. math::

>   CSMRisk_{ENN} = \frac{\text{deaths due to cause C in the ENN age group}}{\text{live births}}

Reviewer 2: The fact that this refers to "cause C" suggests that the "C" in "CSMRisk" is a variable representing an actual cause, e.g., for sepsis we might write "sepsisSMRisk". Is that the intention? Or is the "C" in CSMR still just an abbreviation for the word "cause"? In which case maybe in line 215 it would be good to define the C used here, e.g., change "and" to "and for a given cause C," or something.

>   CSMRisk_{LNN} = \frac{\text{deaths due to cause C in the LNN age group}}{\text{live births} - \text{deaths due to all causes in the ENN age group}}

Reviewer 2: ```suggestion
  \text{CSMRisk}_\text{LNN} = \frac{\text{deaths due to cause C in the LNN age group}}{\text{live births} - \text{deaths due to all causes in the ENN age group}}
```

      - Definition
      - Value or source
      - Note
    * - enn_all_cause_death_count
      - Death count in the early neonatal age group
      - GBD: source='codcorrect', metric_id=1, cause_id=294
      - 
    * - lnn_all_cause_death_count
      - Death count in the early neonatal age group due to all causes
      - GBD: source='codcorrect', metric_id=1, cause_id=294
      - 
    * - enn_cause_specific_death_count
      - Count of deaths due to cause C in the early neonatal age group
      - GBD: source='codcorrect', metric_id=1
      - 
    * - lnn_cause_specific_death_count
      - Count of deaths due to cause C in the late neonatal age group
      - GBD: source='codcorrect', metric_id=1
      - 
    * - live_birth_count
      - Count of live births
      - GBD: covariate_id = 1106
      - 
    * - enn_amr
      - all-cause mortality risk in the early neonatal age group
      - enn_all_cause_death_count / live_birth_count
      - 

>     * - lnn_amr

Reviewer 2: As noted above, we should either change `enn_amr` and `lnn_amr` to be consistent with the above notation, or change the above notation to be consistent with these.

      - all-cause mortality risk in the late neonatal age group
      - lnn_all_cause_death_count / (live_birth_count - enn_all_cause_death_count)

>       - 

Reviewer 2: Seems like it could be good to add a row for `AMR`, defined as _either_ `enn_amr` or `lnn_amr` (or the other notation if we go with that), since `AMR` is what we actually use in our equations.

      - GBD: source='codcorrect', metric_id=1, cause_id=294
      - 
    * - enn_cause_specific_death_count
      - Count of deaths due to cause C in the early neonatal age group
      - GBD: source='codcorrect', metric_id=1
      - 
    * - lnn_cause_specific_death_count
      - Count of deaths due to cause C in the late neonatal age group
      - GBD: source='codcorrect', metric_id=1
      - 
    * - live_birth_count
      - Count of live births
      - GBD: covariate_id = 1106
      - 
    * - acmrisk_enn
      - all-cause mortality risk in the early neonatal age group
      - enn_all_cause_death_count / live_birth_count
      - 
    * - acmrisk_enn
      - all-cause mortality risk in the late neonatal age group
      - lnn_all_cause_death_count / (live_birth_count - enn_all_cause_death_count)
      - 
    * - csmrisk_enn
      - Cause-specific mortality risk in the early neonatal age group
      - enn_cause_specific_death_count / live_birth_count
      - 
    * - csmrisk_lnn
      - Cause-specific mortality risk in the late neonatal age group
      - lnn_cause_specific_death_count / (live_birth_count - enn_all_cause_death_count)
      - 

>     * - :math:`ACMRisk`

Reviewer 2: ```suggestion
    * - :math:`\text{ACMRisk}`
```

      - All-cause mortality risk
      - either acmrisk_enn or acmrisk_lnn depending on the simulant's age group
      - 

>     * - :math:`CSMRisk`

Reviewer 2: ```suggestion
    * - :math:`\text{CSMRisk}`
```
